sdk/eventgrid/azure-eventgrid/samples/preview_samples.md

```diff
@@ -0,0 +1,185 @@
+# EventGrid Client Examples:
+
```

Comment:
is this a new readme structure?/ should this be named README.md?

Reply:
should this file also include the previous info about how to install/etc.?

Reply:
This was my gist of all the samples, I can remove it or change it into a readme